salty-trolls-react/src/App.js

```diff
@@ -1,16 +1,16 @@
-import React, { Component } from 'react';
-import { Route } from 'react-router-dom';
-import axios from 'axios';
+import React, { Component } from "react";
+import { Route } from "react-router-dom";
+import axios from "axios";
 
 //components
-import Navigation from './components/Navigation/Navigation';
-import HackerList from './components/HackerList/HackerList';
-import Login from './components/authentication/Login';
-import Logout from './components/authentication/Logout';
-import Register from './components/authentication/Register';
-import UserPage from './components/UserPage/UserPage';
-import HackerProfile from './components/HackerProfile/HackerProfile';
-import './App.css';
+import Navigation from "./components/Navigation/Navigation";
```

Comment:
I think we have different prettier settings. I use single quotes and your prettier uses double quotes. I am going to change my settings to double quotes as well. 🙂